src/components/ProjectSettingsTabs/VariablesTab/getFormSchema.ts

```diff
@@ -0,0 +1,100 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { FormState } from "./types";
+import { VariableRow } from "./VariableRow";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      varsArray: {
+        type: "array" as "array",
+        items: {
+          type: "object" as "object",
+          properties: {
+            varName: {
+              type: "string" as "string",
+              title: "Variable Name",
+            },
+            varValue: {
+              type: "string" as "string",
+              title: "Variable",
+            },
+            isPrivate: {
+              type: "boolean" as "boolean",
+              title: "Private",
```

Comment:
Are we planning to add the "tooltip explaining what private means" in the tooltips ticket?

Reply:
I didn't see any language for this in the [language Google Doc](https://docs.google.com/document/d/16o6CeDF2Z_7S9SZNouN00Spuio-pF8628zlIPLz34oU/edit), is this correct @ablack12?

Reply:
Good catch -- I think we can use the same description as the legacy ("Users must be logged in to see this project.")

Reply:
Good catch -- should be something like "Private variables have redacted values on the Project Page and the API and cannot be modified" (although does that make it sound like they also can't be deleted?)

Reply:
@ablack12 Hmmm perhaps slightly. Maybe "Private variables have redacted values on the Project Page and the API and cannot be _updated_"?

Reply:
Yes I think that's better!